### PR TITLE
DATA-1008 Changes for drone-ecs-task-update plugin

### PR DESCRIPTION
**Pull request by mateusz-figiel**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -487,7 +487,7 @@ resource "aws_security_group" "ecs_sg_internal" {
 # Create user for drone.io
 
 module "drone-io" {
-  source     = "git::https://github.com/BerlingskeMedia/bm.terraform-module.drone-io?ref=0.9.0"
+  source     = "git::https://github.com/BerlingskeMedia/bm.terraform-module.drone-io?ref=0.9.1"
```
Comment (bmkrra) [inline main.tf:490]: remove the https part, it should only be git@github....

Comment (mateusz-figiel) [reply main.tf:490]: Are you sure? Every module uses this https transport AFAIR...

Comment (bmkrra) [reply main.tf:490]: Taken from other modules like ecs 

module "ecs" {
  source          = "git@github.com:BerlingskeMedia/bm.terraform-module.ecs"
  igw_id          = local.igw_id

Comment (mateusz-figiel) [reply main.tf:490]: But this is taken from manifest, not the module...

Comment (mateusz-figiel) [reply main.tf:490]: Examples:
Module: https://github.com/BerlingskeMedia/bm.terraform-module.drone-io/blob/master/main.tf
Module: https://github.com/BerlingskeMedia/bm.terraform-module.ecs-service/blob/05cac2577af0f44be1457859e61f0cd4f16fd543/main.tf#L120

Comment (mateusz-figiel) [reply main.tf]: OK, I changed transport proto to SSH.

**Review by Maciej-Gierwatowski — APPROVED** [review]

**Review by bmkrra — APPROVED** [review]

**Pull request merged**